apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -20,17 +20,15 @@ class Comment extends Component {
     viewAsTeacher: PropTypes.bool
   };
 
-  state = {isShowingCommentOptions: false};
-
-  onDelete = () => {
-    this.setState({isShowingCommentOptions: false});
-    this.props.onDelete();
+  state = {
+    hideWhileResolved: true
```

Comment:
nit: this variable name is a little confusing to me. It's basically if the comment is hidden or not, but only a resolved comment can be hidden, correct? Could we rename to isHidden or would that also be confusing?

Reply:
Hmm I agree, not the most straightforward name. I thought about isHidden, but that implies that the comment is hidden while true. That may not be the case for an unresolved comment. Thinking...

Reply:
Maybe `hideResolved` is a bit better?

Reply:
How about something like `isHiddenWhenResolved` or `shouldHideWhenResolved` if that's not too verbose?